cmd/queue/main.go

```diff
@@ -307,8 +328,43 @@ func main() {
 		logger.Fatalw("Failed to parse localhost URL", zap.Error(err))
 	}
 
+	queueProxyL3 := fmt.Sprintf("%s:%d", env.ServingPod, networking.ServiceHTTPPort)
+	zipkinEndpoint, err := zipkin.NewEndpoint(env.ServingPod, queueProxyL3)
```

Comment:
Should all this be done iff tracing is enabled the first place?

Reply:
🤔that seems reasonable to me.  @greghaynes?

Reply:
So are we doing this?

Reply:
Yep, I updated this to check if tracing is enabled.  I was seeing some failed integration tests that I wasn't able to reproduce locally, and wanted to confirm that the check was not the culprit, since I didn't change much between passing & failing tests.